packages/discord.js/src/util/Util.ts

```diff
@@ -542,7 +556,9 @@ class Util extends null {
         }
       })
       .replace(/<#[0-9]+>/g, input => {
-        const mentionedChannel = channel.client.channels.cache.get(input.replace(/<|#|>/g, ''));
+        const mentionedChannel = channel.client.channels.cache.get(
+          input.replace(/<|#|>/g, ''),
+        ) as GuildTextBasedChannel;
```

Comment:
```suggestion
        )!;
```

iirc it returns this type so assertion isn't needed.

Reply:
It doesn't return that type.